templates/layout.html

```diff
@@ -15,14 +15,16 @@
     <div class="header">
         <nav>
             <ul class="nav nav-pills pull-right">
-                    {% if user %}
-                <li role="presentation" class="active">
-                    <a href="/#" class="not-link">{{ user.username }}</a>
-                <li role="presentation"><a href="/logout">Log Out</a>
-                    {% else %}
-                <li role="presentation" class="active"><a href="/login">Sign In</a>
-                <li role="presentation"><a href="/join">Sign Up</a>
-                    {% endif %}
+                {% if user %}
+                        {# if user is admin or manager #}
+                        {% if user.role == 2 or user.role == 1 %}
+                            <li role="presentation"><a href="/sample/add">Add learning sample</a>
```

Comment:
Found the proof that it is not necessary: https://www.w3.org/TR/html5/grouping-content.html#the-li-element.